src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/hotshard/HotShardRca.java

```diff
@@ -106,108 +96,151 @@ private void consumeFlowUnit(
                 if (indexName != null && shardId != null) {
                     IndexShardKey indexShardKey = IndexShardKey.buildIndexShardKey(record);
                     double usage = record.getValue(MetricsDB.SUM, Double.class);
```

Comment:
This is per-shard aggregation from the last five seconds. I think by going with the peak instead of avg approach on the analysis level, this value would still make sense being SUM as we are looking at 5-second peaks.